docs/run-keep-node.adoc

```diff
@@ -0,0 +1,359 @@
+
+# TYPE connected_bootstrap_count gauge
+connected_bootstrap_count 10 1623235129569
+
+# TYPE eth_connectivity gauge
+eth_connectivity 1 1623235129789
+```
+
+== Diagnostics
+
+The client exposes the following diagnostics:
+
+- list of connected peers along with their network id and Ethereum operator address,
+- information about the client's network id and Ethereum operator address.
+
+Diagnostics can be enabled in the configuration `.toml` file. It is possible to customize port at which
+diagnostics endpoint is exposed.
+
+Example diagnostics endpoint call result:
+```
+$ curl localhost:8081/diagnostics
+{
+  "client_info" { 
+   "ethereum_address":"0xDcd4199e22d09248cA2583cBDD2759b2acD22381",
+   "network_id":"16Uiu2HAkzYFHsqbwt64ZztWWK1hyeLntRNqWMYFiZjaKu1PZgikN"
+  },
+  "connected_peers": [
+    {"ethereum_address":"0x3712C6fED51CECA83cA953f6FF3458f2339436b4","network_id":"16Uiu2HAkyYtzNoWuF3ULaA7RMfVAxvfQQ9YRvRT3TK4tXmuZtaWi"},
+    {"ethereum_address":"0x4bFa10B1538E8E765E995688D8EEc39C717B6797","network_id":"16Uiu2HAm9d4MG4LNrwkFmugD2pX7frm6ZmA4vE3EFAEjk7yaoeLd"}, 
+    {"ethereum_address":"0x650A9eD18Df873cad98C88dcaC8170531cAD2399","network_id":"16Uiu2HAkvjVWogUk2gq6VTNLQdFoSHXYpobJdZyuAYeoWD66e8BD"},
+    ...
+  ]
+}
+```
+
+[#testnet]
+== icon:flask[] Testnet
+
+The Keep Network Testnet environment is running against Ethereum Görli Testnet.
+To run the client connected to the Testnet add `--goerli` flag to the start command.
+
+A Docker image for Testnet is published under:
+`us-docker.pkg.dev/keep-test-f3e0/public/keep-client:latest` +
+See <<testnet-docker>> section for details.
+
+// TODO: Add faucet details once ready.
+// === Faucet
+
+[#testnet-bootstrap,Testnet Bootstrap Nodes]
+=== Bootstrap Nodes
+
+A client running on testnet establishes connection to the following bootstrap nodes:
+
+[source]
+----
+include::../config/_peers/goerli[]
+----
+
+[#testnet-contracts,Testnet Contracts]
+=== Contracts
+
+[%header,cols="1m,1m"]
+|===
+|Contract
+|Address
+
+|TokenStaking
+|link:https://goerli.etherscan.io/address/0x69f962a0fbA5635e84eC94131f9072108E2E4F24#code[0x69f962a0fbA5635e84eC94131f9072108E2E4F24]
+
+|RandomBeacon
+|link:https://goerli.etherscan.io/address/0x2bA82903B635a96154A515488d2952E86D6adc3A#code[0x2bA82903B635a96154A515488d2952E86D6adc3A]
+
+|WalletRegistry
+|link:https://goerli.etherscan.io/address/0x2363cc10b7680000C02E4a7067A68d1788ffc86F#writeProxyContract[0x2363cc10b7680000C02E4a7067A68d1788ffc86F]
+
+|===
+
+=== Installation
+
+[#testnet-docker,Docker Testnet]
+==== Docker
+
+To get the Docker image run one of the following commands:
+
+* *Latest:* `us-docker.pkg.dev/keep-test-f3e0/public/keep-client`
+
+* *Tagged:* `us-docker.pkg.dev/keep-test-f3e0/public/keep-client:<tag-version>`
+
+=== Run
+
+==== Docker
+
+This is a sample run command for illustration purposes only:
+
+[source,shell,role=small]
+----
+include::resources/docker-start-testnet-sample[]
+----
+
+== Kubernetes
```

Comment:
Is this section important? I would drop it.

Reply:
1020f4842a2ed07df0cebc8cf57e0893f5c6dc45